### PR TITLE
feat: add strict moveType

### LINKED ISSUE #189: New move type - perMove(tentative)
## Description
There's a demand to move panels strictly by `n` panels, following old v2.x's behavior.

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#189 

## Details
This resolves #189

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +36,11 @@
...
+  <a href="https://github.com/naver/egjs-flicking/blob/master/packages/ngx-flicking/README.md"><img width="45" src="https://naver.github.io/egjs-flicking/img/icons/angular.svg" alt="AngularJS" /></a>&nbsp;&nbsp;
+  <a href="https://github.com/naver/egjs-flicking/blob/master/packages/react-flicking/README.md"><img width="45" src="https://naver.github.io/egjs-flicking/img/icons/react.svg" alt="React" /></a>&nbsp;&nbsp;
+  <a href="https://github.com/naver/egjs-flicking/blob/master/packages/vue-flicking/README.md"><img width="45" src="https://naver.github.io/egjs-flicking/img/icons/vue.svg" alt="Vue.js" /></a>&nbsp;&nbsp;
+  <a href="https://github.com/naver/egjs-flicking/blob/master/packages/preact-flicking/README.md"><img width="45" src="https://naver.github.io/egjs-flicking/img/icons/preact.svg" alt="Preact" /></a>&nbsp;&nbsp;
+  <a href="https://github.com/naver/egjs-flicking/blob/master/packages/svelte-flicking/README.md"><img width="45" src="https://naver.github.io/egjs-flicking/img/icons/svelte.svg" alt="Svelte" /></a>
```
Comment (mixed) [inline README.md:43]: There seems to be no svelte image. So, the image is not currently exposed in the svelte readme, but it needs to be checked.

File: src/control/StrictControl.ts [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+   * |---|---|
+   * |{@link ERROR_CODE POSITION_NOT_REACHABLE}|주어진 패널이 제거되었거나, Camera의 {@link Camera#range range} 밖에 있을 경우|
+   * |{@link ERROR_CODE NOT_ATTACHED_TO_FLICKING}|{@link Control#init init}이 이전에 호출되지 않은 경우|
+   * |{@link ERROR_CODE ANIMATION_INTERRUPTED}|사용자 입력에 의해 애니메이션이 중단된 경우|
+   * |{@link ERROR_CODE STOP_CALLED_BY_USER}|발생된 이벤트들 중 하나라도 `stop()`이 호출된 경우|
+   *
+   * </ko>
+   * @return {Promise<void>} A Promise which will be resolved after reaching the target position<ko>해당 좌표 도달시에 resolve되는 Promise</ko>
+   */
+  public async moveToPosition(position: number, duration: number, axesEvent?: OnRelease) {
```
Comment (mixed) [inline src/control/StrictControl.ts:168]: That method contains a lot of logic, what do you think about separating it?

**Review by mixed — APPROVED** [review]
LGTM

File: src/control/states/DraggingState.ts [hunk 3]
```diff
@@ -40,8 +40,12 @@ class DraggingState extends State {
 
     const camera = flicking.camera;
     const prevPosition = camera.position;
+    const position = axesEvent.pos[AXES.POSITION_KEY];
```
Comment (daybrush) [inline src/control/states/DraggingState.ts:43]: Is the onChange code of DraggingState and AnimatingState the same, is it difficult to commonize?

**Review by daybrush — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (WoodNeck) [reply README.md:43]: That will be visible after the release :)

File: src/control/states/DraggingState.ts (hunk 3 above)
Comment (WoodNeck) [reply src/control/states/DraggingState.ts:43]: Right, looks like they're identical. I should deal with that.

File: src/control/StrictControl.ts (hunk 2 above)
Comment (WoodNeck) [reply src/control/StrictControl.ts:178]: Well, I'll try

**Pull request merged**